### PR TITLE
Consume new message UpdateProvidesSourceDistroMessage

### PR DESCRIPTION
**Pull request by pacospace**
## Related Issues and Dependencies

Related-To: https://github.com/thoth-station/thoth-application/issues/507

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: consumer.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 from thoth.messaging import SolvedPackageMessage
 from thoth.messaging import UnresolvedPackageMessage
 from thoth.messaging import UnrevsolvedPackageMessage
+from thoth.messaging import UpdateProvidesSourceDistroMessage
```
Comment (harshad16) [inline consumer.py:41]: this seems to be missing in https://github.com/thoth-station/messaging/blob/b2952d2abfb811c1724bb95b38caf137e501ab66/thoth/messaging/__init__.py
would give us module not found issue.

Comment (pacospace) [reply consumer.py:41]: very nice catch @harshad16!! do you want to take care?

Comment (harshad16) [reply consumer.py:41]: You can take it. 
Would be a short change and a release of messaging.

Comment (pacospace) [reply consumer.py:41]: https://github.com/thoth-station/messaging/pull/284 sure!

Comment (pacospace) [reply consumer.py:41]: new release of messaging added

**Review by harshad16 — APPROVED** [review]
lgtm :+1:

**Pull request merged**